VirtoCommerce.CustomerModule.Web/Scripts/customer.js

```diff
@@ -187,4 +212,12 @@ angular.module(moduleName, [])
               }]
           }
       });
+
+      // predefine search filters for customer search
+        predefinedSearchFilters.register(1583235535540, 'customerSearchFiltersDate', [
```

Comment:
What does this number mean - 1583235535540?

Reply:
Last time modification date in milliseconds

Reply:
The named constant created